Makefile

```diff
@@ -209,10 +208,9 @@ build/ffmpeg-mp4/ffmpeg.bc: $(MP4_SHARED_DEPS)
 		--enable-libmp3lame \
 		--enable-libx264 \
 		--extra-cflags="-s USE_ZLIB=1 -I../lame/dist/include" \
-		--extra-ldflags="-L../lame/dist/lib" \
+		--extra-ldflags="-r -L../lame/dist/lib" \
 		&& \
-	emmake make -j && \
-	cp ffmpeg ffmpeg.bc
+	emmake make -j EXESUF=.bc
```

Comment:
When `emconfigure` is use `emmake` is not necessary.     All the emscripten-specific details will be encoding into the autoconf-generated makefile.  `emmake` is for normal non-autoconf-based Makefiles

Reply:
What should this line be changed to then?